relay-metrics/src/aggregator.rs

```diff
@@ -65,6 +65,7 @@ struct BucketKey {
     metric_name: MetricName,
     tags: BTreeMap<String, String>,
     extracted_from_indexed: bool,
+    randomness: u16,
```

Comment:
We could gate this field with a compile-time flag to reduce the memory footprint in production. Not sure if it's worth it.